common_benchmark_suite/openxla/benchmark/def_types.py

```diff
@@ -44,6 +44,9 @@ class ModelArtifactType(Enum):
   XLA_HLO_DUMP = "xla_hlo_dump"
   STABLEHLO_MLIR = "stablehlo_mlir"
   LINALG_MLIR = "linalg_mlir"
+  TFLITE_FP32 = "tflite_fp32"
+  TFLITE_INT8 = "tflite_int8"
+  TFLITE_UINT8 = "tflite_uint8"
```

Comment:
What will be the problem if we only have `TFLITE`? (or `TOSA_MLIR`)

Reply:
Removed.

Reply:
From an offline discussion, I think it is fine to define in this way.

Ultimately, we should have a structure to define the parameters to export models (so we can specify to export into `[fp32, int8, uint8]` TFLite models and possibly with other parameters). But it should be easy to change once the requirements actually pop up.

Maybe on the enum add comments to explain what are `TFLITE_FP32`, `TFLITE_INT8`, `TFLITE_UINT8`